frontend/src/util/Utility.ts

```diff
@@ -0,0 +1,19 @@
+import SockJS from "sockjs-client";
+import Stomp from "stompjs";
+
+let stompClient:any = null;
+
+export const Connect = (endpoint:string) => {
```

Comment:
Miniscule nit: rename non-component functions to lower-case.

Reply:
Done!